src/components/managementPanel/ManagementPanel.jsx

```diff
@@ -0,0 +1,202 @@
+import { useEffect, useState } from 'react';
```

Comment:
about imports, its a good practice to follow below import order (group separated by one white line each):
1) Imports from npm packages
2) Imports from own files (i.e. other components, utils etc.)
3 Imports from style files (here: scss files)

This pattern in used I every (I hope) our react components